BBox2D bbox
float32 x
float32 y
float32 z
float32 w
float32 h
float32 l
float32 a
